resources/db/pierone.sql

```diff
@@ -74,6 +74,21 @@ SELECT ssd_url AS url,
 ORDER BY created DESC
  LIMIT 1;
 
+-- name: create-or-update-scm-source-data!
+WITH scm_source_data_update AS (
```

Comment:
Good point, but current RDS instance runs 9.4. I'd rather not upgrade it right now.